### ISSUE TITLE
centos package fails automated tests - "which: command not found"

### ISSUE THREAD

**Issue by jessereynolds** [flapjack/omnibus-flapjack#80]
```
/etc/init.d/flapjack: line 21: which: command not found
Error: flapjack isn't in PATH.
Refusing to do anything!
```

**Comment by jessereynolds** [comment]
As seen in:

15:00 <roberta> Failed build of Flapjack v1.3.0rc2 for centos 6 package (experimental) in 19m 37s, log: http://is.gd/vwYUgB

**Comment by Hobbsee** [comment]
Fixed in fe597fee537305373a98e43ea67dc05a03990c91, this is a duplicate of #79

**Comment by Hobbsee** [comment]
Looks like this actually comes from flapjack itself.  Fixed in [maint/1.x f4ab416] Use type instead of which for better cross-platform support
